### PR TITLE
[Android] android-ci.yml

### LINKED ISSUE #5917: [Android] Update CI pipeline to run UI Tests

### PR DESCRIPTION
**Pull request by golddove**
# Related Issue

Fixes #5917 

# Description

Convert Android CI from classic pipeline to yml pipeline

# Sample Card

N/A

# How Verified

Manual pipeline run

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6088)

### CODE REVIEW

File: .pipelines/android-ci.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  inputs:
+    targetType: inline
+    script: >-
+      #!/usr/bin/env bash
+
+      # Install AVD files
+      echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-27;google_apis;x86'
+
+      # Create emulator
+      echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-27;google_apis;x86' --force
```
Comment (almedina-ms) [inline .pipelines/android-ci.yml:57]: super nit: as this PR targets android and not xamarin we could rename the created virtual device to android_emulator

**Review by almedina-ms — APPROVED** [review]

**Comment by MeAdarshMishra** [comment]
Create a new yml file then export old to new then check again

File: .pipelines/android-ci.yml (hunk 1 above)
Comment (golddove) [reply .pipelines/android-ci.yml:57]: Good idea, thanks!

**Review by MeAdarshMishra — COMMENTED** [review]
Yes maybe this works

**Review by paulcam206 — APPROVED** [review]

File: .pipelines/android-ci.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    - main
+    - feature/*
+    - release/*
+
+trigger:
+  branches:
+    include:
+    - main
+    - feature/*
+    - release/*
```
[the remark this answers is not in the document]
Comment (golddove) [reply .pipelines/android-ci.yml:15]: The triggering branch needs to have this yml filed checked in, so should be good!

**Pull request merged**